syntax = "proto3";

package TencentJarvis;

message CommonRequest {
    string version = 1;
    string request_id = 2;
    string user_id = 3;
    string session_id = 4;
    string trace_id = 5;
}

message CommonResponse {
    string version = 1;
    sint32 ret_code = 2;
    string ret_message = 3;
}